src/styles.js

```diff
@@ -13,7 +13,7 @@ export const GlobalStyle = createGlobalStyle`
 
 	body {
 		margin: 0;
```

Comment:
Are you overwriting something here or could you actually delete **margin** and **padding**? (for the sake of cleaner code)

Reply:
Yes, thanks! There was a preset of margin, but no padding